package/skelcd-control-SLED.spec

```diff
@@ -27,7 +27,10 @@
 #
 ######################################################################
 
-Name:           skelcd-control-SLED
+%define         skelcd_name SLED
+%define         skelcd_datadir %{_datadir}/installation-products
```

Comment:
Um, this will be the same for all skelcd packages, right? Would it make sense to share this macro definition with all skelcd packages? Then we would need to touch only one place when changing this and have cleaner skelcd definitions...

We have [macros.yast](https://github.com/yast/yast-devtools/blob/master/build-tools/rpm/macros.yast) in devtools package but we should not depend on it here. But we possibly we could add a similar macro file to the [yast2-installation-control](https://github.com/yast/yast-installation-control) which is already used here.

Reply:
Good to know, thank you! I'll take a look.